nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core-api/src/main/java/org/apache/nifi/connectable/Connectable.java

```diff
@@ -311,5 +311,5 @@ default boolean isSessionBatchingSupported() {
 
     void setMaxBackoffPeriod(String maxBackoffPeriod);
 
-    String evaluateSchedulingPeriodFromParameter(String schedulingPeriod);
+    String evaluateParameter(String parameter);
```

Comment:
As commented, based on the way this is used, I think the signature should be `String evaluateParameters(String value);`